examples/tutorial/jupyter/execution/pandas_on_dask/test/test_notebooks.py

```diff
@@ -0,0 +1,118 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import os
+import sys
+
+import nbformat
+
+UTILS_PARENT_DIR = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), *[".." for _ in range(2)])
+)
+sys.path.insert(0, UTILS_PARENT_DIR)
+from test.utils import (  # noqa
```

Comment:
Why noqa?

Reply:
`noqa` is needed to avoid `E402 module level import not at top of file` flake8 error since we are using `sys.path.insert` prior of utils import.

Reply:
I see. Let's use `noqa: E402` explicitly then.

Reply:
Fixed.